### PR TITLE
Activity and Category image uri fixes

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
@peterp ready for review

**Comment by justinvdm** [comment]
(but depends on #112)

File: scripts/fixtures.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
+export const CATEGORY_IMAGE = 'http://8d5ebfb8.ngrok.io/category-image.png';
```
Comment (peterp) [inline scripts/fixtures.js]: I'm guessing these probably shouldn't be here?

**Comment by peterp** [comment]
👍

**Pull request merged**